semgrep-core/src/matching/Generic_vs_generic.ml

```diff
@@ -1642,7 +1642,8 @@ and m_wildcard (a1, a2) (b1, b2) =
 and m_keyword_attribute a b =
   match (a, b) with
   (* equivalent: quite JS-specific *)
-  | G.Var, (G.Var | G.Let | G.Const) -> return ()
+  | G.Var, (G.Var | G.Let | G.Const) ->
```

Comment:
If I'm not mistaken, if you have `a = G.Var` and `b = G.Var` and `let_is_var` is disabled, this will fail.
It should be:
```ocaml
| G.Var, G.Var -> return ()
| G.Var, (G.Let | G.Const) -> if_config ...
```

Reply:
oh good call!